App.tsx

```diff
@@ -16,17 +16,44 @@ import { StyleSheet } from 'react-native';
 const App = () => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const [mispelling, setMispelling] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const OnPressHandler = () => {
```

Comment:
```suggestion
  const handlePress = () => {
```

Passando pra convenção que usamos. Atenção ao case (`pascalCase`).

Reply:
Boa! bem lembrado